src/test/java/com/plaid/client/integration/AssetReportGetTest.java

```diff
@@ -63,6 +63,16 @@ public void testAssetReportGetSuccess() throws Exception {
       .get(0).getName());
   }
 
+  private boolean containsTransactionWithName(AssetReportGetResponse.AssetReport assetReport) {
+    List<AssetReportGetResponse.Account> accounts = assetReport.getItems().get(0).getAccounts();
+    for (AssetReportGetResponse.Account account : accounts) {
+      if (account.getTransactions().size() > 0) {
+        return account.getTransactions().get(0).getName() != null;
```

Comment:
Please confirm that a `""` transaction name should return `true`.

Also, I can't see this method being used anywhere, what is it meant for?

Reply:
ah it was to prevent some flakiness but yes "" should return true, a report without insights will not have the name key